charts/promitor-agent-scraper/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 0.2.0
+version: 0.3.0
```

Comment:
No need to bump this, this is done in CD

Reply:
And reverted.